x/profiles/keeper/msg_server_dtag_transfers.go

```diff
@@ -154,32 +153,29 @@ func (k MsgServer) AcceptDTagTransferRequest(goCtx context.Context, msg *types.M
 
 		senderAcc := k.ak.GetAccount(ctx, add)
 		if senderAcc == nil {
-			senderAcc = authtypes.NewBaseAccountWithAddress(add)
+			senderAcc = k.ak.NewAccountWithAddress(ctx, add)
```

Comment:
It would be a problem using `NewBaseAccountWithAddress` since its account number is always 0.
To solve it, `accountKeeper.NewAccountWithAddress` is the proper way to create an account